store/instance.go

```diff
@@ -165,7 +165,7 @@ func (s *Store) CountInstanceGroupByEngineAndEnvironmentID(ctx context.Context,
 	rows, err := tx.PTx.QueryContext(ctx, `
 		SELECT engine, environment_id, COUNT(*)
 		FROM instance
-		WHERE row_status = $1
+		WHERE row_status = $1 AND updater_id != 1
```

Comment:
Limit scope to instance ID <= 102 just in case?

Reply:
Good idea. Updated, PTAL